src/expr/term_conversion_proof_generator.h

```diff
@@ -85,7 +98,9 @@ class TConvProofGenerator : public ProofGenerator
    */
   TConvProofGenerator(ProofNodeManager* pnm,
                       context::Context* c = nullptr,
-                      TConvPolicy pol = TConvPolicy::FIXPOINT);
+                      TConvPolicy pol = TConvPolicy::FIXPOINT,
+                      TConvCachePolicy cpol = TConvCachePolicy::NEVER,
```

Comment:
in the documentation above add `cpol`. Also say why the defaults for `pol` and `cpol` are these.

Reply:
You did not explain the defaults.

Reply:
Done